extensions/import/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "import",
   "displayName": "SQL Server Import",
   "description": "SQL Server Import for Azure Data Studio supports importing CSV or JSON files into SQL Server.",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
please note that we have to produce the vsix for this extension by using the `vsce package` locally for now.